internal/cluster/provisioners/gke/provisioner.go

```diff
@@ -128,6 +129,8 @@ func (e GKE) createVarFile() (err error) {
 			buffer.WriteString(fmt.Sprintf("instance_type = \"%v\"\n", np.InstanceType))
 			buffer.WriteString(fmt.Sprintf("max_pods = %v\n", np.MaxPods))
 			buffer.WriteString(fmt.Sprintf("volume_size = %v\n", np.VolumeSize))
+			buffer.WriteString(fmt.Sprintf("preemptible = %v\n", np.SpotInstance))
```

Comment:
Be consistent...

Reply:
@angelbarrera92, you asked for a `soft interface` many times, saying that was the added value that SIGHUP put into. I just tried to comply with your requests keeping the same value in both the cloud providers. In the terraform of GKE, I have to maintain the ubiquitous language that google chooses for the name of that kind of instance.
This is why I opted for keeping in the yaml the same `spotInstance` name for both providers (because is a level of indirection) while in the underneath implementation of the terraform I kept the domain right name for both providers.

Reply:
Sure!
What I don't agree with is using different attribute names on the `terraform modules`. It triggers exactly these confusions/doubts...

Reply:
the terraform code is on the domain side of the cloud provider, so for respecting the domain ubiquitous language, I must put the exact name that the cloud provider involves.

Reply:
You may be right, but if you notice, there are already attributes with standard (our standard) names in terraform modules... like:

- subnetworks: on AWS are subnet-ids while on GKE is a different concept closer to `availability zones`.
- additional_firewall_rules: On AWS are security groups while on GKE are firewall rules.

Then, when we need some specific attributes on gke (for example) we decided to go with the same attribute name from the upstream module plus the `gke_` prefix.

Example: https://github.com/sighupio/furyctl/blob/master/data/provisioners/cluster/gke/variables.tf#L76-L80

TL;DR: What I want is consistency, that's all.